environment.yml

```diff
@@ -30,14 +30,14 @@ dependencies:
   - blosc
   - botocore>=1.11
   - boto3
-  - bottleneck>=1.2.0
+  - bottleneck>=1.2.1
   - fastparquet>=0.2.1
   - html5lib
   - ipython>=5.6.0
   - ipykernel
   - jinja2
   - lxml
-  - matplotlib>=2.0.0
+  - matplotlib>=2.2.2
   - nbsphinx
   - numexpr>=2.6.8
```

Comment:
this seems too high of a version here for numexpr compared to the above

Reply:
This was added in #24015 and does not affect the lowest supported version